src/LanguageServer/Test/HoverTests.cs

```diff
@@ -93,7 +93,7 @@ import datetime
             var hs = new HoverSource(new PlainTextDocumentationSource());
 
             AssertHover(hs, analysis, new SourceLocation(3, 2), "module datetime*", new SourceSpan(3, 1, 3, 9));
-            AssertHover(hs, analysis, new SourceLocation(3, 11), "class datetime*", new SourceSpan(3, 9, 3, 18));
```

Comment:
Something was wrong with the prefix? It helps to tell class vs module.

Reply:
We were going to remove it wholesale for consistency, but I think I have a middle solution that would work slightly better.